packages/admin-api-client/package.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "@shopify/admin-api-client",
+  "version": "0.0.1",
+  "description": "Shopify Admin API Client - A lightweight JS client to interact with Shopify's Admin API",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/shopify-api-js.git"
+  },
+  "author": "Shopify",
+  "license": "MIT",
+  "main": "./dist/umd/admin-api-client.min.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/admin-api-client.d.ts",
+  "exports": {
+    ".": {
+      "module": {
+        "types": "./dist/ts/index.d.ts",
+        "default": "./dist/index.mjs"
```

Comment:
Do we want to have `require` entries for the CJS build? I think that would be the right setup here?

Reply:
I think that's happening a few lines down - let me know if I'm missing something!